packages/client/tests/e2e/_utils/run.ts

```diff
@@ -109,12 +109,12 @@ async function main() {
     }
 
     if (args['--skipPack'] !== true) {
-      await $`cd ${clientPkgPath} && pnpm pack --pack-destination ${__dirname}/../`
-      await $`cd ${cliPkgPath} && pnpm pack --pack-destination ${__dirname}/../`
-      await $`cd ${enginesPkgPath} && pnpm pack --pack-destination ${__dirname}/../`
-      await $`cd ${debugPkgPath} && pnpm pack --pack-destination ${__dirname}/../`
-      await $`cd ${generatorHelperPkgPath} && pnpm pack --pack-destination ${__dirname}/../`
-      await $`cd ${wpPluginPkgPath} && pnpm pack --pack-destination ${__dirname}/../`
+      await $`cd ${clientPkgPath} && pnpm pack --pack-destination /tmp/`
+      await $`cd ${cliPkgPath} && pnpm pack --pack-destination /tmp/`
+      await $`cd ${enginesPkgPath} && pnpm pack --pack-destination /tmp/`
+      await $`cd ${debugPkgPath} && pnpm pack --pack-destination /tmp/`
+      await $`cd ${generatorHelperPkgPath} && pnpm pack --pack-destination /tmp/`
+      await $`cd ${wpPluginPkgPath} && pnpm pack --pack-destination /tmp/`
```

Comment:
As mentioned in the title of the PR, this simplifies and makes running e2e tests much faster because we don't spend time copying the packaged client/cli/debug/generator to each test anymore. It also enables to be able to run tests locally again, and speed things up even more by creating lock files (in a future PR).